portal/src/app/components/home/components/experiment/components/toggles/Toggles.tsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import { ToggleButton } from "../../../../../../shared/components/toggle-button";
+import { EXPERIMENT_EN } from "../../translate/en";
+
+export interface TogglesProps {
+  currentSection: string;
+  handleButtonClick: (section: string) => void;
+}
+
+export const Toggles: React.FC<TogglesProps> = (props: TogglesProps) => {
+  return (
+    <div>
+      <ToggleButton 
```

Comment:
you can use map loop instead and over your tabs array and create for each one of the item ToggleButton 

map(item => {<ToggleButton isSelected={...} >))